lib/application/crowdaction/subscription/subscription_bloc.dart

```diff
@@ -8,21 +8,32 @@ import '../../../domain/crowdaction/crowdaction.dart';
 import '../../../domain/crowdaction/i_crowdaction_repository.dart';
 
 part 'subscription_bloc.freezed.dart';
+
 part 'subscription_event.dart';
+
 part 'subscription_state.dart';
 
 @injectable
 class SubscriptionBloc extends Bloc<SubscriptionEvent, SubscriptionState> {
   final ICrowdActionRepository _crowdActionRepository;
 
   SubscriptionBloc(this._crowdActionRepository)
-      : super(const SubscriptionState.initial()) {
+      : super(SubscriptionState.initial()) {
```

Comment:
Seems like this can still be `const` right?

Reply:
Nope it's a getter function

Reply:
Ah yes, I just saw it returned a `const` and I didn't think about more about it.